### ISSUE TITLE
Huobi (CCXT Pro) - watchOrders and watchBalance never receive data

### ISSUE THREAD

**Issue by jayburston** [ccxt/ccxt#18860]
### Operating System

Linux

### Programming Languages

PHP

### CCXT Version

4.0.57

### Description

After subscribing to watchOrders or watchBalance, no data is ever received.

watchBalance receives an initial balance after subscribing, but never any updates.

### Code

```
  

```

**Comment by carlosmiei** [comment]
Hello again @jayburston, this seems related to the other issue, and I've also tried and seemed to work:

```
 ph huobi watchBalance   
PHP v8.2.1
CCXT version :4.0.57
huobi->watchBalance()
Array
(
    [timestamp] => 
    [datetime] => 
    [info] => Array
        (
            [currency] => usdt
            [accountId] => 44234548
            [balance] => 23.324467200715990285
            [available] => 3.324467200715990243
            [changeType] => 
            [accountType] => trade
            [changeTime] => 
            [seqNum] => 553
        )

    [USDT] => Array
        (
            [free] => 3.324467200716
            [used] => 20
            [total] => 23.324467200716
        )

    [free] => Array
        (
            [USDT] => 3.324467200716
        )

    [used] => Array
        (
            [USDT] => 20
        )

    [total] => Array
        (
            [USDT] => 23.324467200716
        )

)

Array
(
    [timestamp] => 
    [datetime] => 
    [info] => Array
        (
            [currency] => ltc
            [accountId] => 44234548
            [balance] => 0.202815086067214761
            [available] => 0.102815086067214761
            [changeType] => order.place
            [accountType] => trade
            [seqNum] => 147
            [changeTime] => 1691760659488
        )

    [USDT] => Array
        (
            [free] => 19.097739421183
            [used] => 2.0E-18
            [total] => 19.097739421183
        )

    [free] => Array
        (
            [USDT] => 19.097739421183
            [BTC] => 6.150257778E-8
            [DOGE] => 0.08
            [XRP] => 19.96
            [LTC] => 0.10281508606721
            [ADA] => 4.5920051346986
        )

    [used] => Array
        (
            [USDT] => 2.0E-18
            [BTC] => 0
            [DOGE] => 0
            [XRP] => 0
            [LTC] => 0.1
            [ADA] => 0
        )

    [total] => Array
        (
            [USDT] => 19.097739421183
            [BTC] => 6.150257778E-8
            [DOGE] => 0.08
            [XRP] => 19.96
            [LTC] => 0.20281508606721
            [ADA] => 4.5920051346986
        )

    [BTC] => Array
        (
            [free] => 6.150257778E-8
            [used] => 0
            [total] => 6.150257778E-8
        )

    [DOGE] => Array
        (
            [free] => 0.08
            [used] => 0
            [total] => 0.08
        )

    [XRP] => Array
        (
            [free] => 19.96
            [used] => 0
            [total] => 19.96
        )

    [LTC] => Array
        (
            [free] => 0.10281508606721
            [used] => 0.1
            [total] => 0.20281508606721
        )

    [ADA] => Array
        (
            [free] => 4.5920051346986
            [used] => 0
            [total] => 4.5920051346986
        )

)

Array
(
    [timestamp] => 
    [datetime] => 
    [info] => Array
        (
            [currency] => ltc
            [accountId] => 44234548
            [balance] => 0.102815086067214761
            [available] => 0.102815086067214761
            [changeType] => order.match
            [accountType] => trade
            [seqNum] => 148
            [changeTime] => 1691760659494
        )

    [USDT] => Array
        (
            [free] => 19.097739421183
            [used] => 2.0E-18
            [total] => 19.097739421183
        )

    [free] => Array
        (
            [USDT] => 19.097739421183
            [BTC] => 6.150257778E-8
            [DOGE] => 0.08
            [XRP] => 19.96
            [LTC] => 0.10281508606721
            [ADA] => 4.5920051346986
        )

    [used] => Array
        (
            [USDT] => 2.0E-18
            [BTC] => 0
            [DOGE] => 0
            [XRP] => 0
            [LTC] => 0
            [ADA] => 0
        )

    [total] => Array
        (
            [USDT] => 19.097739421183
            [BTC] => 6.150257778E-8
            [DOGE] => 0.08
            [XRP] => 19.96
            [LTC] => 0.10281508606721
            [ADA] => 4.5920051346986
        )

    [BTC] => Array
        (
            [free] => 6.150257778E-8
            [used] => 0
            [total] => 6.150257778E-8
        )

    [DOGE] => Array
        (
            [free] => 0.08
            [used] => 0
            [total] => 0.08
        )

    [XRP] => Array
        (
            [free] => 19.96
            [used] => 0
            [total] => 19.96
        )

    [LTC] => Array
        (
            [free] => 0.10281508606721
            [used] => 0
            [total] => 0.10281508606721
        )

    [ADA] => Array
        (
            [free] => 4.5920051346986
            [used] => 0
            [total] => 4.5920051346986
        )

)

```

**Comment by jayburston** [comment]
Seems to be working fine now, probably my environment only problem.
